app/kafka/manager/ActorModel.scala

```diff
@@ -116,6 +118,7 @@ object ActorModel {
   case class KSGetBrokerState(id: String) extends  KSRequest
 
   case class TopicList(list: IndexedSeq[String], deleteSet: Set[String]) extends QueryResponse
+  case class TopicListWithMoreInfo(list: IndexedSeq[TopicIdentity], deleteSet: Set[String]) extends QueryResponse
```

Comment:
Since this is only used inside KafkaMananger, let's keep it out of ActorModel and in its own file or inside KafkaManager.scala.  This way the ActorModel only consists of classes used by actors.